crates/sargon/src/wrapped_radix_engine_toolkit/high_level/manifest_building/account_locker/manifest_account_locker.rs

```diff
@@ -0,0 +1,151 @@
+use crate::prelude::*;
+use radix_engine_interface::blueprints::locker::{
+    ACCOUNT_LOCKER_CLAIM_IDENT, ACCOUNT_LOCKER_CLAIM_NON_FUNGIBLES_IDENT,
+};
+
+impl From<AccountAddress> for ScryptoComponentAddress {
+    fn from(value: AccountAddress) -> ScryptoComponentAddress {
+        ScryptoComponentAddress::new_or_panic(value.node_id().0)
+    }
+}
+
+impl TransactionManifest {
+    pub fn account_locker_claim(
+        locker_address: &LockerAddress,
+        claimant: &AccountAddress,
+        claimable_resources: Vec<AccountLockerClaimableResource>,
+    ) -> Self {
+        let mut builder = ScryptoManifestBuilder::new();
+        let claimant_arg: ScryptoComponentAddress = (*claimant).into();
+        let claimable_resources =
+            Self::build_claimable_batch(claimable_resources, 50);
```

Comment:
what happens if `claimable_resources.len() > 50`? Should this method not be throwing then? Seems we "truncate" the rest of the resources silently?

Reply:
or perhaps at least return a `bool` (or rather an enum!) telling if ALL resources were claimed or if we did truncate some?

Reply:
We truncate the rest of the resources silently and consider the operation successful without notifying the users in any way (at least for now). From UX perspective it will look like this: the user sees that there are some resources to claim for a specific account, they perform the claim transaction -> then they can do that again until they no longer see the notification that something is claimable.

Reply:
right but if we return a `bool` indicating that we KNOW that more resources can be claimed, then we can do one less network call from hosts right? :) 

i.e. we can continue to show - or show again - the prompt to claim, based on that bool :)

Reply:
Good point

Reply:
Although I agree it makes sense to tell the hosts whether there are more resources to claim or not, I doubt we need it, so most likely we won't use it in the hosts (at least for now - as per current requirements). We'll still have to perform the network calls as there might be other claimable resources besides the ones already known, we can't rely solely on the `bool` from the previous claiming. The question is if it's worth adding a new uniffi exported model for the sake of correctness and for the eventuality in which it will be needed? Or maybe it's better we add it when we know it's needed for sure?